.github/workflows/pr-ui-web.yaml

```diff
@@ -0,0 +1,108 @@
+name: pr-ui-web
+
+on:
+  pull_request:
+    branches:
+      - main
+    paths:
+      - "ui/web-v2/**"
+      - "!**/**.md"
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.ref }}
+  cancel-in-progress: true
+
+env:
+  NODE_VERSION: 16
+  PROTOC_VERSION: 3.18.1
+  BUILDIFIER_VERSION: 5.1.0
+  WEB_DIRECTORY: "ui/web-v2"
+
+jobs:
+  install-dependencies:
+    defaults:
+      run:
+        working-directory: ${{ env.WEB_DIRECTORY }}
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+        with:
+          node-version: ${{ env.NODE_VERSION }}
+      - uses: actions/cache@v3
+        id: yarn-cache
+        with:
+          path: |
+            **/node_modules
+            **/.eslintcache
+            ${{ steps.yarn-cache-dir-path.outputs.dir }}
```

Comment:
You may have forgotten to add a step whose id is `yarn-cache-dir-path`.

Reply:
@masaaania good catch, thanks!
I have fixed it.
You can see the results here:
https://github.com/bucketeer-io/bucketeer/actions/runs/3179533843

Reply:
@cre8ivejp 

Looks good to me about `pr-ui-web.yaml`!!
Are you middle in work about `pr-go.yaml`?

Reply:
I pushed it by accident.
I will delete it, thanks!

Reply:
@masaaania 
I have deleted it. Please take a look at it tomorrow.